src/lib/InstantSearch.ts

```diff
@@ -42,7 +55,25 @@ function defaultCreateURL() {
  * @fires Instantsearch#render This event is triggered each time a render is done
  */
 class InstantSearch extends EventEmitter {
-  constructor(options) {
+  public client: InstantSearchOptions['searchClient'];
+  public insightsClient: InsightsClient | null;
+  public indexName: string;
+  public helper: AlgoliaSearchHelper | null;
+  public mainHelper: AlgoliaSearchHelper | null;
+  public mainIndex: Index;
+  public started: boolean;
+  public templatesConfig: object;
+  public _stalledSearchDelay: number;
+  public _searchStalledTimer: any;
```

Comment:
```suggestion
  public _searchStalledTimer?: number;
```

Reply:
it's not a number in Node, and TS gets confused about this (won't accept number, since we have @types/node)